### ISSUE TITLE
Can't have DT and NC as mods allowed.

### ISSUE THREAD

**Issue by conquestace** [jramseygreen/osu_bot_framework-v3#8]
Match will always abort when you select DT and NC. (Some DT lobbies people want to play NC basically and no way to make it so you can have either mods allowed)

**Comment by jramseygreen** [comment]
Yes you're right... I'll create a more robust mod system for version 4. Thanks for taking the time to post the issue!
